problems/1.types+operators.js

```diff
@@ -2,19 +2,20 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+//function isNumber() {}
+const isNumber = (val) => {
+  return (typeof val === "number") ? true : false;
```

Comment:
Looks good. Could even just return the condition statement. 
Also no need to wrap condition in ()